src/components/formFields/select/select.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import * as styles from "./select.module.css";
+import { Control, Controller, FieldValues } from "react-hook-form";
+import ReactSelect from "react-select";
+import { IReactHookFormProps } from "../types";
+
+export interface ISelectValue {
+  label: string;
+  value: string;
+}
+
+interface ISelectProps {
+  control: Control<FieldValues, any>;
```

Comment:
This type is throwing an incompatibility error when implementing. For now, you can make the type `any`. I'll be making a JIRA issue.